src/helpers/config-helper.js

```diff
@@ -158,13 +158,13 @@ const api = {
 
   urlStringToConfigHash (urlString) {
     try {
-      const urlParts = url.parse(urlString);
+      const urlParts = url.parse(urlString, true);
       let result   = {
         database: urlParts.pathname.replace(/^\//,  ''),
         host:     urlParts.hostname,
         port:     urlParts.port,
         protocol: urlParts.protocol.replace(/:$/, ''),
-        ssl:      urlParts.query ? urlParts.query.indexOf('ssl=true') >= 0 : false
+        ssl:      (urlParts.query && urlParts.query.ssl === 'true')
```

Comment:
`urlParts.query && urlParts.query.ssl === 'true' || false`

Will need to add test for this change as well